### PR TITLE
6694 remove raw queries from connected account

### LINKED ISSUE #6694: Remove raw queries from connected account
## Scope & Context

Remove raw queries from connected account

## Technical inputs

Remove `packages/twenty-server/src/modules/connected-account/repositories/connected-account.repository.ts` and replace it with `TwentyORM`.

### PR DESCRIPTION
**Pull request by bosiraphael**
Closes #6694

### CODE REVIEW

**Comment by lucasbordeau** [comment]
Farewell raw queries ...

File: packages/twenty-server/src/modules/calendar/common/query-hooks/calendar-query-hook.module.ts [hunk 1]
```diff
@@ -4,15 +4,11 @@ import { ObjectMetadataRepositoryModule } from 'src/engine/object-metadata-repos
...
-import { ConnectedAccountWorkspaceEntity } from 'src/modules/connected-account/standard-objects/connected-account.workspace-entity';
 import { WorkspaceMemberWorkspaceEntity } from 'src/modules/workspace-member/standard-objects/workspace-member.workspace-entity';
 
 @Module({
   imports: [
-    ObjectMetadataRepositoryModule.forFeature([
-      ConnectedAccountWorkspaceEntity,
-      WorkspaceMemberWorkspaceEntity,
-    ]),
+    ObjectMetadataRepositoryModule.forFeature([WorkspaceMemberWorkspaceEntity]),
```
Comment (charlesBochet) [inline packages/twenty-server/src/modules/calendar/common/query-hooks/calendar-query-hook.module.ts:11]: do we still need WorkspaceMemberWorkspaceEntity?

File: packages/twenty-server/src/modules/calendar/calendar-event-import-manager/calendar-event-import-manager.module.ts [hunk 2]
```diff
@@ -25,13 +25,11 @@ import { CalendarCommonModule } from 'src/modules/calendar/common/calendar-commo
...
 import { ConnectedAccountModule } from 'src/modules/connected-account/connected-account.module';
 import { RefreshAccessTokenManagerModule } from 'src/modules/connected-account/refresh-access-token-manager/refresh-access-token-manager.module';
-import { ConnectedAccountWorkspaceEntity } from 'src/modules/connected-account/standard-objects/connected-account.workspace-entity';
 import { WorkspaceMemberWorkspaceEntity } from 'src/modules/workspace-member/standard-objects/workspace-member.workspace-entity';
 
 @Module({
   imports: [
     ObjectMetadataRepositoryModule.forFeature([
-      ConnectedAccountWorkspaceEntity,
       BlocklistWorkspaceEntity,
```
Comment (charlesBochet) [inline packages/twenty-server/src/modules/calendar/calendar-event-import-manager/calendar-event-import-manager.module.ts:33]: do we still need the other entities?

File: packages/twenty-server/src/engine/object-metadata-repository/metadata-to-repository.mapping.ts [hunk 3]
```diff
@@ -1,13 +1,11 @@
 import { BlocklistRepository } from 'src/modules/blocklist/repositories/blocklist.repository';
-import { ConnectedAccountRepository } from 'src/modules/connected-account/repositories/connected-account.repository';
 import { AuditLogRepository } from 'src/modules/timeline/repositiories/audit-log.repository';
 import { TimelineActivityRepository } from 'src/modules/timeline/repositiories/timeline-activity.repository';
 import { WorkspaceMemberRepository } from 'src/modules/workspace-member/repositories/workspace-member.repository';
 
 export const metadataToRepositoryMapping = {
   AuditLogWorkspaceEntity: AuditLogRepository,
   BlocklistWorkspaceEntity: BlocklistRepository,
```
Comment (charlesBochet) [inline packages/twenty-server/src/engine/object-metadata-repository/metadata-to-repository.mapping.ts:8]: do we still use the other ones?

**Review by charlesBochet — APPROVED** [review]
LGTM!

**Pull request merged**